### PR TITLE
Create readme.md

### PR DESCRIPTION
**Pull request by EvanBerner**
readme for home page of rtd

### CODE REVIEW

File: docs/readme.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+Project Jellyfish
+=======
+
+[![Code Climate](https://codeclimate.com/repos/551958df69568055c00020f8/badges/b227e3bde5577507002e/gpa.svg)](https://codeclimate.com/repos/551958df69568055c00020f8/feed)
+[![Test Coverage](https://codeclimate.com/repos/551958df69568055c00020f8/badges/b227e3bde5577507002e/coverage.svg)](https://codeclimate.com/repos/551958df69568055c00020f8/feed)
+[![Build Status](https://travis-ci.org/projectjellyfish/api.svg?branch=master)](https://travis-ci.org/projectjellyfish/api)
```
Comment (chriskacerguis) [inline docs/readme.md]: Should these be here?

File: docs/readme.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+**Getting Started (Basic Requirements)**
+
+Jellyfish has the following requirements:
+
+* Ruby (Please check .ruby-version for the latest version to use, generally, this is the current version of Ruby)
+* PostgreSQL 9.4+
+* Red Hat Enterprise Linux / CentOS
+
+To see a detailed list of [System Requirements](https://github.com/projectjellyfish/api/wiki/System-Requirements), please check out our [Wiki](https://github.com/projectjellyfish/api/wiki).
```
Comment (chriskacerguis) [inline docs/readme.md]: Do we want to change these links to the correct RTD links?

File: docs/readme.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+**How to Install Project Jellyfish**
+
+Jellyfish is a Ruby on Rails application.  Please choose the appropriate installation guide for your system:
+
+* Mac Users: [INSTALL-OSX.md](https://github.com/projectjellyfish/api/blob/master/INSTALL-OSX.md) - Mac OS Installation (generally used for development)
+
+* Windows Users: [INSTALL-RHEL.md](https://github.com/projectjellyfish/api/blob/master/INSTALL-RHEL.md)  - Red Hat Enterprise Linux / CentOS installation
```
Comment (chriskacerguis) [inline docs/readme.md]: Should we move all the install guides to RTD?

File: docs/readme.md (hunk 2 above)
Comment (sonejah21) [inline docs/readme.md]: Yes, we do. This is currently in the works - we are adding the System Requirements file to the "/docs" folder so it syncs into RTD.

File: docs/readme.md (hunk 3 above)
Comment (sonejah21) [inline docs/readme.md]: Yes again! All install guides will be moved into the RTD "/docs" file. We'll figure out the nesting of children files like these soon so it is organized how we'd like.

**Pull request merged**